src/mrpro/operators/_FourierOp.py

```diff
@@ -32,9 +32,7 @@ def __init__(
         recon_shape: SpatialDimension[int],
         encoding_shape: SpatialDimension[int],
         traj: KTrajectory,
-        oversampling: SpatialDimension[float] = SpatialDimension(
-            2.0, 2.0, 2.0
-        ),  # TODO: maybe not the nicest, since only used for nuFFT
+        oversampling: float | SpatialDimension[float] = 2.0,  # only used for nuFFT
```

Comment:
Could we also have this just as a float. I cannot think of any application where we would need different oversampling factor along different directions...